### PR TITLE
Add ThemeProvider and prop-types dependency.

### PR DESCRIPTION
**Pull request by floridamaniac**
Related tickets:
* https://trello.com/c/vZB1rxoO
* https://trello.com/c/IXdgsJ9F

Added `prop-types` dependency required by ESLint in order to make `ThemeProvider` work without syntax errors. Created light scaffolding for a theme to be used as a starting point that can be expanded on later.

Screen Shot 2022-04-01 at 12 37 30 PM

### CODE REVIEW

File: client/src/components/Overview/Overview.jsx [hunk 1]
```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import styled from 'styled-components';
 
 function Overview() {
   return (
-    <h1>hello world: Overview</h1>
+    <OverviewSection>
```
Comment (ryand8008) [inline client/src/components/Overview/Overview.jsx:6]: Styled components! Love it

**Review by ryand8008 — APPROVED** [review]
Thanks for setting up the styled components for each section

**Review by theVikingMan — APPROVED** [review]

**Comment by ivyhu630** [comment]
Screen Shot 2022-04-01 at 3 58 38 PM
This is probably a user error. I can seem to run client-dev. and eslint is giving me a warning on line 4 of the import styled in index.jsx

**Review by ivyhu630 — APPROVED** [review]
Style components  looking good

**Pull request merged**